pkg/authentication/handlers.go

```diff
@@ -253,6 +254,26 @@ func (a *API) handleLogout(w http.ResponseWriter, r *http.Request) {
 	a.cookieManager.ClearAccessTokenCookie(w)
 	a.cookieManager.ClearRefreshTokenCookie(w)
 
+	session, err := a.identities.GetIdentitySession(ctx, r.Cookies())
+	if err != nil {
+		a.logger.Errorf("failed to retrieve kratos session, err: %v", err)
+		a.badRequest(w, err)
+		return
+	}
+	if session == nil {
+		a.badRequest(w, fmt.Errorf("failed to get session id"))
+		return
+	}
```

Comment:
nitpick: feel free to disregard, the cases handled here seem to be pretty much equivalent, we could just sum it up in one single case (we can still print the error if it happens to be nil), for example (take this only as an example)

```suggestion
	if err != nil || session == nil {
	        err = fmt.Errorf("failed to retrieve kratos session or session not found, err: %v", err)
		a.logger.Error(err.Error())
		a.badRequest(w, err)
		return
	}
```

Reply:
I splitted those in case session is nil but there is no error, I'd prefer to keep it